modin/engines/ray/generic/io.py

```diff
@@ -41,6 +41,6 @@ def func(df):
             df.to_sql(**kwargs)
             return pandas.DataFrame()
 
-        result = qc._modin_frame._fold_reduce(1, func)
+        result = qc._modin_frame._apply_full_axis(1, func, new_index=[], new_columns=[])
```

Comment:
Why is `new_index=[]` and not `None`? Same question for `new_columns`

Reply:
We know that the result of applying `func` will be an empty DataFrame, so instead of extra twice calling `get_indices` in `_apply_full_axis` we specify new indices explicitly here